script/binary-controller

```diff
@@ -18,13 +18,13 @@ set -e
 
 if [ -z "$1" ]; then
 #    TODO: Skip windows at this moment
-    OS_PLATFORM_ARG=(-os="darwin linux")
```

Comment:
here I agree, because it is the controller. But for the CLI we do need OSX and windows builds